php/commands/post.php

```diff
@@ -465,6 +465,15 @@ private function read_from_file_or_stdin( $arg ) {
 		}
 		return file_get_contents( $readfile );
 	}
+
+	/**
+	 * Preserve text content by escaping backslashes that get unescaped by wp_unslash() down the line.
+	 * @param string $str  The string to be escaped
+	 * @return string  The escaped string
+	 */
+	public static function escape($str) {
```

Comment:
And, rather than running `str_replace()` against all fields, let's just run it against the two applicable fields: `post_excerpt` and `post_content`